app/com/linkedin/drelephant/tunin/TuningInput.java

```diff
@@ -0,0 +1,385 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.linkedin.drelephant.tunin;
+
+import models.TuningAlgorithm;
+
+/**
+ * This class holds the parameters passed to rest api from the client.
+ */
+public class TuningInput {
+
+    private String flowDefId;
```

Comment:
to maintain consistency in convention, please prefix underscore to all variables.

Reply:
done